### PR TITLE
Added int offset

### PR DESCRIPTION
**Pull request by yahgwai**

### CODE REVIEW

File: src/express-lane-auction/RoundTimingInfo.sol [hunk 1]
```diff
@@ -88,7 +100,7 @@ library RoundTimingInfoLib {
         pure
         returns (uint64, uint64)
     {
-        uint64 roundStart = info.offsetTimestamp + info.roundDurationSeconds * round;
+        uint64 roundStart = uint64(info.offsetTimestamp + int64(info.roundDurationSeconds * round));
```
Comment (gzeoneth) [inline src/express-lane-auction/RoundTimingInfo.sol:103]: while all the cast here are unsafe, they are generally safe under the assumption that 1) timestamp is well within the range of uint32 and 2) offset is before timestamp

however, the particular cast is unsafe even with those assumption with e.g. negative offset + round 0

**Review by gzeoneth — APPROVED** [review]
LGTM, commented on an edge case of `roundTimestamps`

File: src/express-lane-auction/RoundTimingInfo.sol (hunk 1 above)
Comment (gzeoneth) [reply src/express-lane-auction/RoundTimingInfo.sol:103]: Anyway I suppose [this](https://github.com/OffchainLabs/nitro-contracts/blob/5f288d76b1fd7cd3dff6960208217d9a982565b7/src/express-lane-auction/IExpressLaneAuction.sol#L246-L249) already explained the limitation of the `roundTimestamps` method that make this specific edge case not a concern,  but maybe we can consider to store the round where the last round change happened and revert `roundTimestamps` if a round before that is requested.

Comment (yahgwai) [reply src/express-lane-auction/RoundTimingInfo.sol:103]: I'll add more documentation about this limitation

**Pull request merged**